### PR TITLE
send notification email on new deposits

### LINKED ISSUE #225: Automatically send Amy e-mail when deposit is complete
Can we have Deep Blue Data send Amy an e-mail when someone uploads anything?

### PR DESCRIPTION
**Pull request by njaffer**
closes #225

### CODE REVIEW

**Comment by njaffer** [comment]
Noticed the email address is not correct, changing it now

File: app/controllers/curation_concerns/generic_works_controller.rb [hunk 1]
```diff
@@ -7,9 +7,16 @@ class CurationConcerns::GenericWorksController < ApplicationController
   include Sufia::WorksControllerBehavior
 
   before_action :check_recent_uploads, only: [:show]
+  after_action  :notify_rdr, only: [:create]
 
   set_curation_concern_type GenericWork
 
+  def notify_rdr
+    @msg = "https://deepblue.lib.umich.edu"+ main_app.curation_concerns_generic_work_path(curation_concern) ;
+    email = WorkMailer.deposit_work(@current_user.email,@msg)
```
Comment (grosscol) [inline app/controllers/curation_concerns/generic_works_controller.rb]: We don't want to be sending email on behalf of the user.  I can see a utility to this, but it's too close to email spoofing.

Comment (njaffer) [inline app/controllers/curation_concerns/generic_works_controller.rb]: sure, should the sender  and receiver be both notification email address

File: app/controllers/curation_concerns/generic_works_controller.rb [hunk 2]
```diff
@@ -7,9 +7,16 @@ class CurationConcerns::GenericWorksController < ApplicationController
   include Sufia::WorksControllerBehavior
 
   before_action :check_recent_uploads, only: [:show]
+  after_action  :notify_rdr, only: [:create]
 
   set_curation_concern_type GenericWork
 
+  def notify_rdr
+    @msg = "https://deepblue.lib.umich.edu"+ main_app.curation_concerns_generic_work_path(curation_concern) ;
```
Comment (grosscol) [inline app/controllers/curation_concerns/generic_works_controller.rb]: The url helper should work here so the protocol and host doesn't have to be hardcoded. `main_app.curation_concerns_generic_work_url`

**Comment by grosscol** [comment]
This looks like it fulfills the letter of the request from #225.  I think this might be better off as a daily digest job. Calling this after create might create a lot of individual emails for @amyneeser or whomever the recipient ends up being.  That can be a subsequent request from ResearchData if they want it.

**Pull request merged**